desktop/main/walletFile.ts

```diff
@@ -270,6 +270,55 @@ export const copyWalletFile = async (filePath: string, outputDir: string) => {
   return newFilePath;
 };
 
+export const isWalletFileValid = async (filePath: string) => {
+  try {
+    const wallet = await loadRawWallet(filePath);
+
+    if (typeof wallet?.meta !== 'object' || typeof wallet.crypto !== 'object') {
+      return false;
+    }
+
+    const { crypto, meta } = wallet;
+
+    const isWalletFileFieldsValid =
+      typeof meta.displayName === 'string' &&
+      typeof meta.created === 'string' &&
+      typeof meta.remoteApi === 'string' &&
+      typeof meta.type === 'string' &&
+      typeof crypto.cipher === 'string' &&
+      typeof crypto.cipherText === 'string';
```

Comment:
Move this into `shared/types/guards.ts` module and name function `isWalletMeta`.